content/pages/sphinx.html

```diff
@@ -0,0 +1,294 @@
+    {% endmarkdown %}
+    {% endcall %}
+
+    {% call marketing.highlight_item(
+        icon="fa-eye",
+        header="Never lose focus with link previews",
+        color="violet") %}
+
+    {# TODO: Add link preview to the `link previews` docs #}
+
+    {% markdown %}
+    Get instant context on every internal link in your docs.
+    With instant link previews, 
+    <strong>skim the content without opening *another* tab</strong>.
+    {% endmarkdown %}
+        {% endcall %}
+    {% endcall %}
+</section>
+
+<div class="ui very padded container">
+
+{{ basic.header_icon(
+  header="Join thousands of happy users",
+  subheader="",
+  icon="purple fa-message-lines",
+  icon_style="--fa-secondary-color: blueviolet;"
+) }}
+
+{# TODO: Turn these into testomonials. #}
+
+{#
+  Cards for an articles or case studies view
+
+  See https://fomantic-ui.com/views/card.html
+#}
+<div class="ui three link stackable cards">
+    <a href="https://flask.palletsprojects.com/en/2.2.x/" class="card">
+    <div class="ui small centered image">
+      <img src="{{ SITE_URL }}/images/flask-logo.png" alt="Flask logo">
+      </div>
+      <div class="content">
+        <div class="header">Flask</div>
+        <div class="meta">
+          Web Framework
+        </div>
+        <div class="description">
+            Flask is a Python web framework built with a small core and easy-to-extend philosophy.
+        </div>
+      </div>
+    </a>
+
+    <a href="https://docs.jupyter.org/en/latest/" class="card">
+    <div class="ui medium centered image">
+      <img src="{{ SITE_URL }}/images/jupyter-logo.png" alt="Jupyter logo">
+      </div>
+      <div class="content">
+        <div class="header">Jupyter</div>
+        <div class="meta">
+          Data Science Environment
+        </div>
+        <div class="description">
+            Jupyter Notebook is a web-based interactive computational environment for creating notebook documents.
+        </div>
+      </div>
+    </a>
+
+    <a href="https://docs.godotengine.org/en/stable/index.html" class="card">
+    <div class="ui small centered image">
+      <img src="{{ SITE_URL }}/images/godot-logo.png" alt="Godot Engine logo">
+    </div>
+      <div class="content">
+        <div class="header">Godot Engine</div>
+        <div class="meta">
+          Game Engine
+        </div>
+        <div class="description">
+          Godot Engine is a free and open source community-driven 2D and 3D game engine.
+        </div>
+      </div>
+    </a>
+</div>
+
+<div class="ui very padded basic vertical segment">
+  <div class="ui horizontal divider">Try it out</div>
+</div>
+
+<div class="ui horizontally container">
+  <div class="ui very padded centered grid">
+    <div class="middle aligned row">
+
+      <h2 class="ui huge header">
+        Everything you need, and nothing you don't.
+      </h2>
+
+      <div class="ten wide computer sixteen wide tablet sixteen wide mobile column">
```

Comment:
This section is awesome by the way, it looks great :100:

Frankly, I think we could lead this page with this content. It does a great job of illustrating our point here -- bringing your Sphinx project to Read the Docs is easy. We do still need to address the reader profile that is arriving at this page without knowledge of Read the Docs at all though.

At very least, I would make this the second/third block of content, and at least putting this above testimonials.

Reply:
> At very least, I would make this the second/third block of content, and at least putting this above testimonials.

I've done this 👍🏼